api/account/api.go

```diff
@@ -112,9 +112,10 @@ func ConvertToAPIAccount(userKc kc.UserRepresentation) AccountRepresentation {
 		var accreds []AccreditationRepresentation
 		for _, accredJSON := range values {
 			var accred AccreditationRepresentation
-			json.Unmarshal([]byte(accredJSON), &accred)
-			accred.Expired = keycloakb.IsDateInThePast(accred.ExpiryDate)
-			accreds = append(accreds, accred)
+			if json.Unmarshal([]byte(accredJSON), &accred) == nil {
```

Comment:
if there is an issue during unmarshalling it would be good to log it as a warning.